test/framework/k8s_cluster.go

```diff
@@ -873,26 +873,17 @@ func (c *K8sCluster) closePortForwards(name string) {
 }
 
 func (c *K8sCluster) deleteCRDs() error {
-	stdout, err := k8s.RunKubectlAndGetOutputE(c.GetTesting(), c.GetKubectlOptions(), "get", "crds", "-o", "yaml")
+	out, err := k8s.RunKubectlAndGetOutputE(c.GetTesting(), c.GetKubectlOptions(), "get", "crds", "-o", "name")
```

Comment:
Huh TIL `-o name`

Reply:
resolved this because I wanted this to get merged to test auto-merge action in KM, hope you don't mind :)

Reply:
I mean yeah, it's meant to be resolved... ;)